Update changelog apr 25 (#444)
OctoMind-dev/mintlify-docs: changelog/changelog.mdx

## March 2025

### 2025-03-30

> - new email flow for testing email flows to sign in, forgot password flow and more. see [email flows](/advanced/email-flows)

Reviewer: ```suggestion
- `Email flow`: test email flows to sign in, forgot password flow and more. See [email flows](/advanced/email-flows)
```

> - new predefined variable: `$OCTO_EMAIL` for testing email flows

Reviewer: ```suggestion
- `Predefined variable` added: `$OCTO_EMAIL` for testing email flows
```

> - new extract value step that allows you to extract values from the page and store them in a dynamic variable

Reviewer: ```suggestion
- New `extract value` step that allows you to extract values from the page and store them in a dynamic variable
```
